nixos/tests/incus/incus-tests.nix

```diff
@@ -4,6 +4,7 @@ import ../make-test-python.nix (
     lib,
 
     lts ? true,
+    appArmor ? false,
```

Comment:
Since you'll be in here anyway, please put this with the other test variants in alphabetical order. Thankd

Reply:
The idea is that AA is more like a general setting like LTS, and not an individual test itself. So we might want to check more combinations later on.

I'll still change this.